platform/node/DEVELOPING.md

```diff
@@ -1,19 +1,71 @@
-# Developing the Mapbox GL Native Node.js module
+# Developing the Maplibre GL Native Node.js module
 
-This document explains how to build the [Node.js](https://nodejs.org/) bindings for [Mapbox GL Native](../../README.md) for contributing to the development of the bindings themselves. If you just want to use the module, you can simply install it via `npm`; see [README.md](README.md) for installation and usage instructions.
+This document explains how to build the [Node.js](https://nodejs.org/) bindings for [Maplibre GL Native](../../README.md) for contributing to the development of the bindings themselves. If you just want to use the module, you can simply install it via `npm`; see [README.md](README.md) for installation and usage instructions.
 
 ## Building
 
 To develop these bindings, you’ll need to build them from source. Building requires the prerequisites listed in either
 the [macOS](../macos/INSTALL.md#requirements) or [Linux](../linux/README.md#prerequisites) install documentation, depending
 on the target platform.
 
+First you'll need to install dependencies:
+
+
+#### MacOS
+
+```bash
+brew list cmake || brew install cmake
+brew list ccache || brew install ccache
+brew list ninja || brew install ninja
+brew list pkgconfig || brew install pkgconfig
+brew list chargepoint/xcparse/xcparse || brew install chargepoint/xcparse/xcparse
+brew list pkg-config || brew install pkg-config
+brew list glfw3 || brew install glfw3
+brew list libuv || brew install libuv
+pip install ansi2html scipy
+```
+
+#### Linux
+
+```bash
+sudo apt-get install -y ca-certificates git gzip tar python3-scipy
```

Comment:
Suggestion: please put each package on a newline:

```bash
sudo apt-get update && sudo apt-get install -y \
    ca-certificates \
    gzip \
    ...
```

Also - some of these packages do not seem strictly necessary?  I'd suggest separating this list into those that are required (build tools, mesa, glfw, etc) vs those that are used only in development (e.g., `valgrind`) vs those that are strictly optional (`fonts-noto`?).  There are a bunch here that I've not had to install into a Docker Debian container in order to build `maplibre-gl-native` (though still may be running into segfaults because of missing deps).

It may be good to note that some of these specific packages are dependent on your version of Linux, and may require a different package version (e.g., g++).